onnxruntime/core/providers/cpu/nn/regex_full_match.cc

```diff
@@ -0,0 +1,36 @@
+#include "regex_full_match.h"
+#include "core/common/common.h"
+
+namespace onnxruntime {
+ONNX_CPU_OPERATOR_KERNEL(
+    RegexFullMatch,
+    20,
+    KernelDefBuilder()
+        .TypeConstraint("T1", DataTypeImpl::GetTensorType<std::string>())
+        .TypeConstraint("T2", DataTypeImpl::GetTensorType<bool>()),
+    RegexFullMatch);
+
+RegexFullMatch::RegexFullMatch(const OpKernelInfo& info) : OpKernel(info) {
+  ORT_ENFORCE(info.GetAttr<std::string>("pattern", &pattern_).IsOK());
+  ORT_ENFORCE(RE2(pattern_).ok(), "Invalid pattern: ", pattern_);
+}
+
+Status RegexFullMatch::Compute(OpKernelContext* context) const {
+  RE2 re(pattern_);
```

Comment:
can we initialize a RE2 data member in the ctor and reuse it instead of creating one every call to Compute()?